configuration/editors.md

```diff
@@ -0,0 +1,188 @@
+---
+layout: documentation
+title: Editors
+---
+
+{% include base.html %}
+
+# Editors - Different ways to power up your textual configuration
```

Comment:
Not sure, up to you. I'd imagine something simpler like "Editors for your Textual openHAB Configuration"